SimTKmath/include/simmath/Optimizer.h

```diff
@@ -309,15 +309,80 @@ class SimTK_SIMMATH_EXPORT OptimizerSystem {
  *      - 2: all files are written to the current directory.
  *      - 3: output to console, and all files are written to the current
  *
+ *
+ * Encoding of Variables
+ *
+ * Inappropriate initialization of the algorithm may lead to resampling of the
+ * parameter distribution. Since some parameters may be defined in a bounded
+ * region [a, b]. The CMA algorithm involves sampling a random distribution for
+ * the variables (parameters) in the optimization problem. If the sampled values
+ * do not lie within the variables' bounds, CMA must resample the distribution
+ * until the variables lie within the bounds. This resampling is undesirable,
+ * and may prevent the algorithm from functioning properly. There are two ways
+ * to avoid excessive resampling:To overcome this the problem the user can
+ * formulate the problem as follows:
+ *
+ * 1) Either define the bounds of the parameter and choose the appropriate stddev
+ * and initial value for each parameter (see init_stepsize)
+ * 2) Or to reformulate the problem, by rescaling the parameter space so that
+ * each parameter map in a region between e.g. [0, 1] and each parameter has an
+ * initial value of 0.5 and stddev of 0.2 (see comments below).
+ *
+ * The specific formulation of a (real) optimization problem has a tremendous
```

Comment:
I would add a note somewhere in this section where you got this from (both for a reference for people. and since this is a bit of copy/paste from the website you linked earlier).